src/Http/Controllers/SwaggerController.php

```diff
@@ -12,18 +12,23 @@
 class SwaggerController extends BaseController
 {
     /**
-     * Dump api-docs.json content endpoint.
+     * Dump api-docs content endpoint. Supports dumping a json, yml or yaml file.
      *
-     * @param string $jsonFile
+     * @param string $file
      *
-     * @return \Illuminate\Http\Response
+     * @return \Response
      */
-    public function docs($jsonFile = null)
+    public function docs(string $file = null)
     {
-        $filePath = config('l5-swagger.paths.docs').'/'.
-            (! is_null($jsonFile) ? $jsonFile : config('l5-swagger.paths.docs_json', 'api-docs.json'));
+        $targetFile = config('l5-swagger.paths.docs_json', 'api-docs.json');
+        $extension = 'json';
+        if (!is_null($file)) {
+            $targetFile = $file;
+            $extension = explode('.', $file)[1];
+        }
+        $filePath = config('l5-swagger.paths.docs').'/'.$targetFile;
 
-        if (config('l5-swagger.generate_always') || ! File::exists($filePath)) {
+        if (! File::exists($filePath)) {
```

Comment:
Why `config('l5-swagger.generate_always')` is removed?

Reply:
That was a mistake I'll add it back in.